vaadin-testbench-core-junit5/src/test/java/com/vaadin/testbench/browser/ParameterizedBrowserTestDisabledOnMultipleBrowsersTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2000-2024 Vaadin Ltd
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ * See <https://vaadin.com/commercial-license-and-service-terms> for the full
+ * license.
+ */
+
+package com.vaadin.testbench.browser;
+
+import java.util.List;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import com.vaadin.testbench.annotations.BrowserConfiguration;
+import com.vaadin.testbench.annotations.RunOnHub;
+import com.vaadin.testbench.parallel.BrowserUtil;
+
+/**
+ * This test should be disabled, since it will run
+ * a @{@link com.vaadin.testbench.ParameterizedBrowserTest} annotated method but
+ * requiring execution on multiple browsers. If it fails with a
+ * {@link NullPointerException} it means the
+ * {@link BrowserExtension#evaluateExecutionCondition(ExtensionContext)} is not
+ * working correctly.
```

Comment:
So the validation passes Does this mean that the implementation is broken as it doesn't throw an exception for multiple used browsers with parameterized test?

Reply:
It is correct that the validation passes because the test is disabled by the addition in `CapabilitiesUtils.evaluateExecutionCondition()`, called by `BrowserExtension.evaluateExecutionCondition()`.
The comment means that if the test throws an exception instead of being disabled, there is a bug in the `evaluateExecutionCondition` implementation.

Reply:
Right, so it disables the test, not throws. Thanks!